package.json

```diff
@@ -54,7 +54,7 @@
   "scripts": {
     "husky:install": "husky install",
     "test": "jest --maxWorkers=4",
-    "test:integration": "jest --config jest.integration.config.ts --maxWorkers=4",
+    "test:integration": "jest --config jest.integration.config.ts --maxWorkers=4 && npm run test:cleanup:reports",
```

Comment:
This will only run the clean-up if the tests all pass. I don't know if that is intended.

It will also change the return code of the whole `npm run ...` command to always be successful (as the clean-up will always be successful) and might break error detection for failing tests in the pipeline.

To preserve the return value of jest you can do:

`jest --config jest.integration.config.ts --maxWorkers=4 ; RET=$? ; npm run test:cleanup:reports ; exit $RET`

Reply:
ah Thanks Peter, updated npm script to preserve the return value and execute cleanup script regardless of test pass/fail